patterns/component-accordion.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Title: Accordion
+ * Slug: meetgoat/component-image
+ * Categories: ubc-component
+ * Viewport Width: 1280
+ * Block Types: core/template-part/UBC Content
+ * Inserter: yes
+ *
+ * @package UBC Collab ISP Child Theme Pattern Library
+ */
+
+?>
+
+<!-- wp:group {"className":"isp-accordion\u002d\u002dsingle","layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
```

Comment:
I've done a search and replace to fix this. 

The cause of this actually appears to be WordPress' formatting. When you add a class using a double hyphen `.block__elem--modifier` wordpress automatically replaces the double hyphen with unicode when you switch to code view. You can update in code view and as you switch back to live view you can see WP switch it back to unicode. A little bit annoying as the WP code standards recommended BEM naming convention (though it doesn't specifically mention the --modifier portion. I'm sure there is a good reason though.

I did a search replace in the theme which fixes it in the theme and WP accepts the patterns, but it does switch it back to unicode characters in the admin but everything works fine. I could easily do a search and replace to change the classes from a double hyphen to something else however that doesn't change the classes anywhere on the site the patterns are already used. I'd either need to manually update the class names (maybe a s&r in the db would work?) or add css to handle both the old and new class names. 

Since everything works both ways I just went with the codebase search and replace back to double hyphens since patterns are unlikely to change much going forward. 

Happy to change the class names entirely if you think thats the better route.